### PR TITLE
Remove secret store files on uninstall, but not update

### PR DESCRIPTION
**Pull request by zanderz**
Tested update and removal scenarios, with both versions of secret key files
Note there is lots of other stuff we still leave behind, but I assume we want to at least leave .mpack files on purpose?

### CODE REVIEW

**Comment by chrisnojima** [comment]
@maxtaco thoughts

**Comment by maxtaco** [comment]
Yes, I think it's the right approach to save the .mpack files but to burn the .ss files.  Better if we "shred" them if that's possible.  There's also the {{.ns2}} files that should be trashed.

File: packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs [hunk 1]
```diff
@@ -260,4 +261,12 @@
               Execute="immediate"
               Return="ignore"/>
   </Fragment>
+  <Fragment>
+    <CustomAction Id="LogoutUser"
+              Directory="INSTALLFOLDER"
+              ExeCommand="[INSTALLFOLDER]keybaserq.exe -wait &quot;[INSTALLFOLDER]keybase.exe&quot; logout"
```
Comment (akalin-keybase) [inline packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs]: logout automatically does the deleting/shredding of the right files, right?

Comment (zanderz) [reply packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs]: Yes, see secret_store_file.go
Tested an installer built from this branch: http://s3.amazonaws.com/prerelease.keybase.io/windows/Keybase_1.0.45-20180326142420%2B531e573.386.exe

**Comment by patrickxb** [comment]
Logout will shred the secret file and the noise file.  And it will shred any version 1 secret store file as well.

**Pull request merged**